e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/validation/broker-postcode-validation.spec.js

```diff
@@ -0,0 +1,216 @@
+    BROKER,
+  },
+} = ROUTES.INSURANCE;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.exporterBusiness;
+
+const BROKER_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+const ERROR_ASSERTIONS = {
+  errorField: broker[POSTCODE],
+  expectedErrorsCount: 5,
+  errorIndex: 4,
+  errorMessageEmpty: BROKER_ERRORS[POSTCODE].IS_EMPTY,
+  errorMessageFormat: BROKER_ERRORS[POSTCODE].INCORRECT_FORMAT,
+};
+
+context('Insurance - Your business - Broker Page - Validation - Postcode', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.navigateToUrl(START);
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitCompanyDetails();
+    cy.completeAndSubmitNatureOfYourBusiness();
+    cy.completeAndSubmitTurnoverForm();
+
+    cy.getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const url = `${Cypress.config('baseUrl')}${ROOT}/${referenceNumber}${BROKER}`;
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe('when the postcode field is left empty', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageEmpty,
+      } = ERROR_ASSERTIONS;
+
+      cy.submitAndAssertFieldErrors(errorField, null, errorIndex, expectedErrorsCount, errorMessageEmpty);
+    });
+  });
+
+  describe('when the postcode has 1 letter in the first part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'S 2AA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has no letters in the first part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = '22 2AA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has 2 digits in the second part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'SW1 22A';
```

Comment:
Could we move all the values into the `examples` directory like we have for phone and website? `phone-number-examples.js`